.github/workflows/reportpackagingfailures.yml

```diff
@@ -0,0 +1,43 @@
+name: Report Build Package Failures
+on:
+  workflow_run:
+    workflows: [Build packages, CI]
```

Comment:
I'm not convinced we should report every time CI fails: it's pretty common for it fail on PRs, and I think notifying each time will add too much noise to the channel. It might be better to only send the notification if it failed when run against the `main` or `staging` branches; the former means our nightly checks failed, while the latter means something failed in Bors

Reply:
using `${{ github.event.workflow_run.conclusion != 'success'  && github.event.workflow_run. event != 'pull_request'}}` in the `if:` on line `10` might do it 🤔

Reply:
@JLockerman The intention is to remove CI failures reporting later. Keeping it there initially so we see how this works for us since we don't build packages very often.

Reply:
@jerryxwu That makes sense, 👍 sound like a plan

Reply:
> using `${{ github.event.workflow_run.conclusion != 'success' && github.event.workflow_run.event != 'pull_request'}}` in the `if:` on line `10` might do it 🤔

Good suggestion. I will make the change to reduce noise.